apps/console/src/features/connections/api/authenticators.ts

```diff
@@ -383,3 +385,32 @@ export const updateFederatedAuthenticator = (
             return Promise.reject(error);
         });
 };
+
+/**
+ * Get federated authenticator details.
+ *
+ * @returns A promise containing the response.
+ */
+export const getFederatedAuthenticatorsList = (): Promise<any> => {
```

Comment:
Should be able to use the type `FederatedAuthenticatorMetaInterface` rather `any` here.